src/SWP/Component/MultiTenancy/Model/OrganizationInterface.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher MultiTenancy Component.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Component\MultiTenancy\Model;
+
+use SWP\Component\Common\Model\EnableableInterface;
+use SWP\Component\Common\Model\SoftDeletableInterface;
+use SWP\Component\Common\Model\TimestampableInterface;
+use SWP\Component\Storage\Model\PersistableInterface;
+
+interface OrganizationInterface extends PersistableInterface, SoftDeletableInterface, EnableableInterface, TimestampableInterface
```

Comment:
no, it doesn't have to. You can have an empty array. It doesnt tell you you must have at least one tenant.